__tests__/intergration.test.js

```diff
@@ -61,6 +61,56 @@ describe("GET: /api/articles", () => {
         });
       });
   });
+  test("200: responds with array of articles filtered by given topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("200: responds with array of articles filtered by multiple given topics", () => {
+    const acceptedCategories = ["mitch", "cats"];
+    return request(app)
+      .get("/api/articles?topic=mitch&topic=cats")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(13);
+        articles.forEach((article) => {
+          expect(acceptedCategories).toContain(article.topic);
+        });
+      });
+  });
+  test("200: returns articles with selected topic ignoring anything else invalid on query", () => {
+    return request(app)
+      .get("/api/articles?category=computer&topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
```

Comment:
This is a nice test but I would adjust the test description so its ignoring any other query as category may be a valid query later on but it just might not be yet so to be future proof I wouldnt call it invalid